scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -138,6 +171,7 @@ class FossId(
         user = fossIdScannerOptions["user"]
             ?: throw IllegalArgumentException("No FossId User configuration found.")
         packageNamespaceFilter = fossIdScannerOptions["packageNamespaceFilter"].orEmpty()
+        queryAuthenticatorForHttpURLs = fossIdScannerOptions["queryAuthenticatorForHttpURLs"].toBoolean()
```

Comment:
Why are these properties called "...HttpURLs"? If I'm not mistaken, you query unconditionally for any protocol, so we should not create the impression it's restricted to HTTP. I'd also use a name that emphasizes the security implications that you mention in the commit message. Something like "addAuthenticationToUrl" or so.